docs/docs_source/build_test_ship/validator.rst

```diff
@@ -0,0 +1,86 @@
+
+..  _validator:
+
+WOPI Validation application
+===========================
+
+To assist hosts in verifying their WOPI implementation, Office Online provides a WOPI Validation application that
+executes a test suite against a host's WOPI implementation. The test suite verifies a variety of things, including that
+the semantics for all of the WOPI operations (:ref:`CheckFileInfo`, :ref:`GetFile`, :ref:`PutFile`, etc.) are correct
+and that request/response headers are set properly. New tests are added regularly.
+
+The WOPI Validation application is an Office Online application similar to Word Online or PowerPoint Online.
+It uses the ``.wopitest`` file extension. The WOPI Validation application is included in the :ref:`discovery`
+XML just like all other Office Online applications.
+
+..  code-block:: xml
+
+    <app name="WopiTest" checkLicense="true">
+        <action name="view" urlsrc="https://onenote.officeapps-df.live.com/hosting/WopiTestFrame.aspx" ext="wopitest"/>
+        <action name="getinfo" urlsrc="https://onenote.officeapps-df.live.com/hosting/GetWopiTestInfo.ashx" ext="wopitest"/>
+    </app>
+
+The validation application provides two :ref:`WOPI Actions`, :wopi:action:`view` and :wopi:action:`getinfo`, which
+can be used to trigger the test suite.
+
+
+Interactive WOPI validation
+---------------------------
+
+The simplest way to use the validation application is to use the *view* action. To use the *view* action hosts should
+treat ``.wopitest`` files the same way other Office documents are treated. In other words, hosts should do the
+following:
+
+#. Launch a :term:`host page` pointed at the ``.wopitest`` file.
```

Comment:
Suggest adding something like "Ideally, this should be the same host page used to host regular Office Online sessions, so that the WOPI Validator can test things like PostMessage and do some validation on the way the frame was loaded."